pages/dkp/kommander/1.3/projects/project-deployments/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Project Deployments
+title: Project Deployments
+menuWeight: 1
+excerpt: Project Deployments can be used to manage GitOps based Continuous Deployments which can automatically be created on all the Kubernetes clusters associated with the Project, in the corresponding namespace.
+---
+
+Kommander Projects can be configured with GitOps based Continuous Deployments for federation of your Applications to associated clusters of the project. This is backed by Dispatch which enables software and applications to be continuously deployed (CD) using GitOps processes. GitOps enables the application to be deployed as per a manifest that is stored in a Git repository. This ensures that the application deployment can be automated, audited and declaratively deployed to the infrastructure.
+
+## What is GitOps?
+
+GitOps is a modern software deployment strategy. The configuration describing how your application is deployed to a cluster are stored in a Git repository. The configuration is continuously synchronized from this Git repository to the cluster, ensuring that the specified state of the cluster always matches what is defined in the "GitOps" Git repository.
```

Comment:
"from this Git repository " would **the** sound better than **this**?

Reply:
Probably, yes.